src/Microsoft.Azure.WebJobs.Logging/Internal/TimeBucket.cs

```diff
@@ -27,5 +29,63 @@ public static long ConvertToBucket(DateTime dt)
             }
             return min;
         }
+
+        // Used to compute epoch. 
+        const long ticksBaseline = 633663648000000000L;
+        const long ticksPerMonth = 26785144663987L;
+        
+        public static DateTime CommonEpoch = new DateTime(0, DateTimeKind.Utc); // maps to epoch 0 
```

Comment:
Used for the list of FunctionDefinitions, which are timeless (they don't have any time stamp information). 

---
In reply to: [82906813](https://github.com/Azure/azure-webjobs-sdk/pull/862#discussion_r82906813) [](ancestors = 82906813)